world/physics/RigidBody.h

```diff
@@ -0,0 +1,127 @@
+#ifndef C___RIGIDBODY_H
+#define C___RIGIDBODY_H
+
+#include "math/EigenWrapper.h"
+
+StateVector f(const StateVector& x, const Vector6& u, const Matrix_3x3& InertiaTensor,
```

Comment:
We should add a docstring explaining that the input u should not include gravity since that is manually added in in the implementation. The same applies to the rk4 function that follows.

Reply:
This function has been changes. 'u' is now a vector of magnetorquer and RW voltages